velox/type/Type.h

```diff
@@ -455,6 +455,10 @@ class Type : public Tree<const TypePtr>, public velox::ISerializable {
 
   virtual bool isPrimitiveType() const = 0;
 
+  virtual bool isOrderable() const = 0;
```

Comment:
here would be the right place for a comment explaining what exactly these two methods should return, semantically.

Reply:
Added.